app/services/wordpress_service.rb

```diff
@@ -55,15 +47,18 @@ def request_access_token
       username: username,
       password: password
     }
-    response = connection.post('oauth2/token', request_body)
+    response = Faraday.post('https://public-api.wordpress.com/oauth2/token', request_body)
     response_body = JSON.parse(response.body)
     raise_invalid_token_error(response_body) if response.status == 400
 
     response_body['access_token']
   end
 
   def connection
-    Faraday.new('https://public-api.wordpress.com/')
+    Faraday.new('https://public-api.wordpress.com/rest/v1.1/') do |conn|
```

Comment:
maybe we should move the API url to a constant or env variable, WDYT?

Reply:
yeah, absolutely!